.promu.yml

```diff
@@ -18,6 +18,8 @@ build:
         -X github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
         -X github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
 tarball:
+    # Whenever there are new files to include in the tarball,
+    # remember to make sure the new files will be generated after `make build`
```

Comment:
Minor but, end with `.`

Reply:
@SuperQ , thanks for reminding. My bad habit! Just updated. Thanks.